### PR TITLE
Added create group & lib/dist folder 

### PR DESCRIPTION
**Pull request by p3tuh**
Mostly complete basic front end. Forms send POST & GET requests with objects. 

Will refactor to use bower once I actually understand how to use bower.

Testing to come soon.

### CODE REVIEW

File: client/app/auth/auth.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ angular.module('boundless.auth', [])
 		Auth.signin($scope.user)
 			.then(function(token) {
 					//this is where we will will set a token/cookie/jwt
-				// $window.localStorage.setItem('our url here', token);
+				$window.localStorage.setItem('our url here', token);
```
Comment (dsernst) [inline client/app/auth/auth.js]: I notice you're using "our url here", but we can actually use any key. The browser already controls by domain, so we can only see the items we set.

File: client/app/services/services.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 angular.module('boundless.services', [])
 
 	//services to fetch & make groups
-.factory('Groups', function ($http) {
+.factory('Groups', function($http, $location, $window) {
```
Comment (dsernst) [inline client/app/services/services.js]: why did we need to inject $location and $window? Are they being used in this factory?

**Comment by dsernst** [comment]
tiny bit concerned about this PR-- note it adds 9200+ lines. Especially if we are going to be adding a bower file anyway, which would remove the need to check all of these libraries into version control.

Just want to make sure we're all cool with what's going on here before we go ahead and do it.

@delucco @p3tuh

File: client/app/services/services.js (hunk 2 above)
Comment (p3tuh) [inline client/app/services/services.js]: hmm... I think it was left over from a copy & paste.

**Comment by p3tuh** [comment]
HA creating bower.json file now. I think I know what I'm doing now....

File: client/app/auth/auth.js (hunk 1 above)
Comment (p3tuh) [inline client/app/auth/auth.js]: Ah I see, any suggestions on what our key should be?

**Comment by p3tuh** [comment]
Fixed these issues. Today I Learned how .bowerrc works and how to untrack folder with git =).

Pushed the changes to my fork so this pull request should automatically reflect those changes.

**Comment by dsernst** [comment]
Niiiiice!

File: client/app/auth/auth.js (hunk 1 above)
Comment (dsernst) [inline client/app/auth/auth.js]: how about "token"

**Comment by p3tuh** [comment]
Crap, missed that one. The first parameter in 
$window.localStorage.setItem is any key we want right? I have the rest 
of them set as 'boundless-avocado' for now.

Best,

Peter Kim
510.604.7443
chong.peter.kim@gmail.com

**Comment by dsernst** [comment]
Yep, any key we want :smile:

**Comment by p3tuh** [comment]
Sweet, committed the changes.

Best,

Peter Kim
510.604.7443
chong.peter.kim@gmail.com

**Comment by dsernst** [comment]
Awesome

**Pull request merged**